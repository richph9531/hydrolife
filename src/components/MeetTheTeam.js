import React from 'react';

import classes from '../styles/MeetTheTeam.module.css';

const MeetTheTeam = props => {
  const TeamMap = props.data.map((item, pos) => (
    <div className={classes.TeamMemberCard} key={pos} >
      <p>{item.firstName} {item.lastName}</p>
      <p>{item.title}</p>
      <p>{item.about}</p>
      <img className={classes.TeamImages}
        src={`${process.env.PUBLIC_URL}/media/team/${item.firstName}${item.lastName}.png`}
        // alt={item.firstName}
        alt=""
        // className={props.data.currentlySelectedTeamMember === item.id ? classes.TeamMemberSelected : classes.TeamMember}
        // onClick={() => props.data.onTeamMemberSelection(pos)}
      />
      <br></br><br></br>
      </div>
    )
  );

  return (
    <div className={classes.MeetTheTeam}>
      <h1>Meet the team</h1>
      <div>
        { TeamMap }
      </div>
      <br></br>
      <br></br>
    </div>
  );
};

export default MeetTheTeam;