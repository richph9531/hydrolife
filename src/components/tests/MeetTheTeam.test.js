import React from 'react';

import { render } from '@testing-library/react';
import MeetTheTeam from '../MeetTheTeam';

import teamData from '../../data/meetTheTeam.json';


test('Meet the Team renders correctly', () => {
  const view = render(<MeetTheTeam  data={teamData}/>);
  expect(view.baseElement).toMatchSnapshot();
});
