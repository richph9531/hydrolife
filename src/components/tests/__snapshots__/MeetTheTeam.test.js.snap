// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meet the Team renders correctly 1`] = `
<body>
  <div>
    <div
      class="MeetTheTeam"
    >
      <h1>
        Meet the team
      </h1>
      <div>
        <div
          class="TeamMemberCard"
        >
          <p>
            Heather
             
            Philips
          </p>
          <p>
            Director
          </p>
          <p>
            Add some content
          </p>
          <img
            alt=""
            class="TeamImages"
            src="/media/team/HeatherPhilips.png"
          />
          <br />
          <br />
        </div>
        <div
          class="TeamMemberCard"
        >
          <p>
            Mike
             
            Oosthuyzen
          </p>
          <p>
            Director
          </p>
          <p>
            Add some content
          </p>
          <img
            alt=""
            class="TeamImages"
            src="/media/team/MikeOosthuyzen.png"
          />
          <br />
          <br />
        </div>
        <div
          class="TeamMemberCard"
        >
          <p>
            Mac
             
            Oosthuyzen
          </p>
          <p>
            Director
          </p>
          <p>
            Add some content
          </p>
          <img
            alt=""
            class="TeamImages"
            src="/media/team/MacOosthuyzen.png"
          />
          <br />
          <br />
        </div>
        <div
          class="TeamMemberCard"
        >
          <p>
            Adien
             
            Oosthuyzen
          </p>
          <p>
            Director
          </p>
          <p>
            Add some content
          </p>
          <img
            alt=""
            class="TeamImages"
            src="/media/team/AdienOosthuyzen.png"
          />
          <br />
          <br />
        </div>
        <div
          class="TeamMemberCard"
        >
          <p>
            Rick
             
          </p>
          <p>
            Senior Manager
          </p>
          <p>
            Add some content
          </p>
          <img
            alt=""
            class="TeamImages"
            src="/media/team/Rick.png"
          />
          <br />
          <br />
        </div>
        <div
          class="TeamMemberCard"
        >
          <p>
            Yvonne
             
          </p>
          <p>
            Accounts
          </p>
          <p>
            Add some content
          </p>
          <img
            alt=""
            class="TeamImages"
            src="/media/team/Yvonne.png"
          />
          <br />
          <br />
        </div>
        <div
          class="TeamMemberCard"
        >
          <p>
            Matthew
             
          </p>
          <p>
            Technician
          </p>
          <p>
            Add some content
          </p>
          <img
            alt=""
            class="TeamImages"
            src="/media/team/Matthew.png"
          />
          <br />
          <br />
        </div>
        <div
          class="TeamMemberCard"
        >
          <p>
            Gary
             
          </p>
          <p>
            Technician
          </p>
          <p>
            Add some content
          </p>
          <img
            alt=""
            class="TeamImages"
            src="/media/team/Gary.png"
          />
          <br />
          <br />
        </div>
        <div
          class="TeamMemberCard"
        >
          <p>
            Logan
             
          </p>
          <p>
            Administration
          </p>
          <p>
            Add some content
          </p>
          <img
            alt=""
            class="TeamImages"
            src="/media/team/Logan.png"
          />
          <br />
          <br />
        </div>
        <div
          class="TeamMemberCard"
        >
          <p>
            Keegan
             
          </p>
          <p>
            Work Husband
          </p>
          <p>
            Add some content
          </p>
          <img
            alt=""
            class="TeamImages"
            src="/media/team/Keegan.png"
          />
          <br />
          <br />
        </div>
      </div>
      <br />
      <br />
    </div>
  </div>
</body>
`;
